OpenRA.Mods.Common/Pathfinder/PathGraph.cs

```diff
@@ -127,17 +127,17 @@ int GetCostToNode(CPos destNode, CVec direction)
 		{
 			int movementCost;
 			if (mobileInfo.CanEnterCell(
-				World as World,
-				Actor as Actor,
+				World,
+				Actor,
 				destNode,
 				out movementCost,
-				IgnoredActor as Actor,
+				IgnoredActor,
 				checkConditions) && !(CustomBlock != null && CustomBlock(destNode)))
 			{
 				return CalculateCellCost(destNode, direction, movementCost);
 			}
 
-			return InvalidNode;
+			return Constants.InvalidNode;
```

Comment:
I think the original refactor PR forgot about that too (or I just overlooked it). Anyway, fixed.